docs/index.html

```diff
@@ -8,7 +8,7 @@
 <meta name="author" content="">
 <meta name="keyword" content="">
 
-    <link rel="shortcut icon" href="/node-mongodb-native/s/img/favicon.png">
```

Comment:
Commenting here, but referring to the img code below:
- `logo-mongodb-header.png`
- `mongodb-university-logo.png`
- `node.png`

Were we replacing these?